pyrato/edc.py

```diff
@@ -1139,3 +1140,68 @@ def truncate_energy_decay_curve(energy_decay_curve, threshold):
         _truncate_energy_decay_curve(energy_decay_curve.time, threshold),
         energy_decay_curve.times,
         energy_decay_curve.comment)
+
+
+def _edc_linregress(energy_decay_curve: pf.TimeData, t0: float, t1: float):
+    """
+    Takes an EDC as input and performs linear regression
+    within a time interval.
+
+    Parameters
+    ----------
+    energy_decay_curve : pyfar.TimeData
+        The energy decay curve
+    t0 : float
+        Start time of the interval
+    t1 : float
+        End time of the interval
+
+    Returns
+    -------
+    slope : float
+        The slope of the linear regression
+    intercept : float
+        The intercept of the linear regression
+    """
+
+    if t0 > t1:
+        raise ValueError("t0 must be smaller than t1")
+
+    t0_idx = energy_decay_curve.find_nearest_time(t0)
+    t1_idx = energy_decay_curve.find_nearest_time(t1)
+
+    if t0_idx == t1_idx:
+        raise ValueError("t0 and t1 must not be equal")
+
+    slope, intercept, _, _, _ = linregress(
+        energy_decay_curve.times[t0_idx:t1_idx],
+        energy_decay_curve.time[t0_idx:t1_idx])
+
+    return slope, intercept
```

Comment:
Im not to familiar with the details, but it might make sence to already return the reverberation_time relative to 60db decay here.